### PR TITLE
OBSDOCS-217-Logging 5.6.7 on 4.10

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.10, 4.11, 4.12

Issue: [OBSDOCS-217](https://issues.redhat.com//browse/OBSDOCS-217)

Link to docs preview: https://drive.google.com/file/d/13lpAD2wXcdFqni6rN6mt1l7_1DkQhGYY/view?usp=drive_link

QE review:
- [x] QE has approved this change.

Additional information:
https://github.com/openshift/openshift-docs/pull/61603
https://github.com/openshift/openshift-docs/pull/61604

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/lgtm

**Comment by libander** [comment]
@anpingli / @kabirbhartiRH / @QiaolingTang - LGTM please

**Comment by kabirbhartiRH** [comment]
Some bugs on 5.6.7 but not having release notes.
https://issues.redhat.com/browse/LOG-4099
https://issues.redhat.com/browse/LOG-4184
https://issues.redhat.com/browse/LOG-4146

https://issues.redhat.com/browse/LOG-4199 - This one has release notes.

**Review by periklis — APPROVED** [review]
LGTM

File: modules/logging-rn-5.6.7.adoc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+= Logging 5.6.7
+This release includes link:https://access.redhat.com/errata/RHBA-2023:3493[OpenShift Logging Bug Fix Release 5.6.7].
+
+[id="openshift-logging-5-6-7-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the LokiStack gateway returned label values for namespaces without applying the access rights of a user. With this update, the LokiStack-gateway applies permissions to label value requests, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3728[LOG-3728])
+
+* Before this update, the `time` field of log messages did not parse as `structured.time` by default in Fluentd when they included a timestamp. With this update, parsed log messages will include a `structured.time` field if the output destination supports it. (link:https://issues.redhat.com/browse/LOG-4090[LOG-4090])
+
+* Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to timeout. With this update, the LokiStack global and per-tenant `queryTimeout` settings affects the route timeout settings, resolving the issue.(link:https://issues.redhat.com/browse/LOG-4130[LOG-4130])
```
Comment (bburt-rh) [inline modules/logging-rn-5.6.7.adoc:14]:
```suggestion
* Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to time out. With this update, the LokiStack global and per-tenant `queryTimeout` settings affects the route timeout settings, resolving the issue.(link:https://issues.redhat.com/browse/LOG-4130[LOG-4130])
```

File: modules/logging-rn-5.6.7.adoc [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+* Before this update, LokiStack CRs with values defined for tenant limits but not global limits caused the Loki Operator to crash. With this update, the Operator is able to process LokiStack CRs with only tenant limits defined , resolving the issue. (link:https://issues.redhat.com/browse/LOG-4199[LOG-4199])
```
Comment (bburt-rh) [inline modules/logging-rn-5.6.7.adoc:16]:
```suggestion
* Before this update, LokiStack CRs with values defined for tenant limits but not global limits caused the Loki Operator to crash. With this update, the Operator is able to process LokiStack CRs with only tenant limits defined, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4199[LOG-4199])
```

File: modules/logging-rn-5.6.7.adoc [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+* Before this update, the {product-title} web console generated errors after an upgrade due to cached files of the prior version retained by the web browser. With this update, these files are no longer cached, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4099[LOG-4099])
+
+* Before this update, Vector generated certificate errors when forwarding to default Loki. With this update, logs can be forwarded to Loki using Vector without errors. (link:https://issues.redhat.com/browse/LOG-4184[LOG-4184])
```
Comment (bburt-rh) [inline modules/logging-rn-5.6.7.adoc:20]:
```suggestion
* Before this update, Vector generated certificate errors when forwarding to the default Loki instance. With this update, logs can be forwarded without errors to Loki by using Vector. (link:https://issues.redhat.com/browse/LOG-4184[LOG-4184])
```

File: modules/logging-rn-5.6.7.adoc [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+* Before this update, the ClusterLogging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the ClusterLogging Operator API will no longer require a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:
```
Comment (bburt-rh) [inline modules/logging-rn-5.6.7.adoc:22]:
```suggestion
* Before this update, the ClusterLogging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the ClusterLogging Operator API no longer requires a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:
```

File: modules/logging-rn-5.6.7.adoc [hunk 5]
```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+// cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-7_{context}"]
+= Logging 5.6.7
+This release includes link:https://access.redhat.com/errata/RHBA-2023:3493[OpenShift Logging Bug Fix Release 5.6.7].
+
+[id="openshift-logging-5-6-7-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the LokiStack gateway returned label values for namespaces without applying the access rights of a user. With this update, the LokiStack-gateway applies permissions to label value requests, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3728[LOG-3728])
```
Comment (mramendi) [inline modules/logging-rn-5.6.7.adoc:10]: "LokiStack gateway" or "LokiStack-gateway"? I'd pick one for the same phrase.

File: modules/logging-rn-5.6.7.adoc [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+* Before this update, the `time` field of log messages did not parse as `structured.time` by default in Fluentd when they included a timestamp. With this update, parsed log messages will include a `structured.time` field if the output destination supports it. (link:https://issues.redhat.com/browse/LOG-4090[LOG-4090])
```
[the remark this answers is not in the document]
Comment (mramendi) [reply modules/logging-rn-5.6.7.adoc:12]: I'd also suggest "when the messages included a timestamp" instead of using "they"

**Review by periklis — COMMENTED** [review]
LGTM

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: modules/logging-rn-5.6.7.adoc [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+= Logging 5.6.7
+This release includes link:https://access.redhat.com/errata/RHBA-2023:3493[OpenShift Logging Bug Fix Release 5.6.7].
+
+[id="openshift-logging-5-6-7-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, the LokiStack gateway returned label values for namespaces without applying the access rights of a user. With this update, the LokiStack gateway applies permissions to label value requests, resolving the issue. (link:https://issues.redhat.com/browse/LOG-3728[LOG-3728])
+
+* Before this update, the `time` field of log messages did not parse as `structured.time` by default in Fluentd when the messages included a timestamp. With this update, parsed log messages will include a `structured.time` field if the output destination supports it. (link:https://issues.redhat.com/browse/LOG-4090[LOG-4090])
+
+* Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to time out. With this update, the LokiStack global and per-tenant `queryTimeout` settings affects the route timeout settings, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4130[LOG-4130])
```
Comment (abrennan89) [inline modules/logging-rn-5.6.7.adoc:14]:
```suggestion
* Before this update, the LokiStack route configuration caused queries running longer than 30 seconds to time out. With this update, the LokiStack global and per-tenant `queryTimeout` settings affect the route timeout settings, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4130[LOG-4130])
```

File: modules/logging-rn-5.6.7.adoc [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+* Before this update, the {product-title} web console generated errors after an upgrade due to cached files of the prior version retained by the web browser. With this update, these files are no longer cached, resolving the issue. (link:https://issues.redhat.com/browse/LOG-4099[LOG-4099])
+
+* Before this update, Vector generated certificate errors when forwarding to the default Loki instance. With this update, logs can be forwarded without errors to Loki by using Vector. (link:https://issues.redhat.com/browse/LOG-4184[LOG-4184])
+
+* Before this update, the ClusterLogging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the ClusterLogging Operator API no longer requires a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:
```
Comment (abrennan89) [inline modules/logging-rn-5.6.7.adoc:22]:
```suggestion
* Before this update, the Cluster Logging Operator API required a certificate to be provided by a secret when the `tls.insecureSkipVerify` option was set to `true`. With this update, the Cluster Logging Operator API no longer requires a certificate to be provided by a secret in such cases. The following configuration has been added to the Operator's CR:
```

**Review by abrennan89 — COMMENTED** [review]
just 2 last comments

**Review by abrennan89 — APPROVED** [review]
/lgtm
🥳

**Pull request merged**